### PR TITLE
(MODULES-3490) Sync from Community Repo

### PR DESCRIPTION
**Pull request by ferventcoder**

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -16,11 +16,17 @@
       default => 'enable'
     }
 
+    if versioncmp($chocolatey::chocolatey_version, '0.9.10.0') >= 0 {
+      $_find_str = "autoUninstaller|${_enable_autouninstaller}d|"
+    } else {
+      $_find_str = "autoUninstaller - [${_enable_autouninstaller}d]"
+    }
```
Comment (ferventcoder) [inline manifests/config.pp]: This actually still has an issue to be resolved.

Comment (glennsarti) [inline manifests/config.pp]: Is this still outstanding?

Comment (ferventcoder) [inline manifests/config.pp]: It is. I think I have this fixed up though so I will commit and update this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +10,31 @@ to the next level. Puppet will rerelease a supported module under the original
 versioning scheme. For now we are using a number less than 1.0 to show that this
 could have some technical issues and should be treated as a prerelease version.
 
+## 2016-06-20 Release 1.2.5
```
Comment (glennsarti) [inline CHANGELOG.md]: I don't think we should be taking in the change log from the community repo e.g. We never released 1.2.4.  I would expect this to appear as a changelog item for our unsupported releases.

File: templates/InstallChocolatey.ps1.erb [hunk 3]
```diff
@@ -1,105 +1,135 @@
...
-if (![System.IO.Directory]::Exists($chocoPkgDir)) { [System.IO.Directory]::CreateDirectory($chocoPkgDir); }
-Copy-Item "$file" "$nupkg" -Force -ErrorAction SilentlyContinue
-
-#}
-#catch
-#{
-#  Write-Host "$($_.Exception.Message)"
-#  exit 1
-#}
+# ==============================================================================
```
Comment (glennsarti) [inline templates/InstallChocolatey.ps1.erb]: Any reason why this is an entire file change instead of a standard line by line diff?  Line endings?

Comment (ferventcoder) [inline templates/InstallChocolatey.ps1.erb]: Unknown. The actual changes here we was the tickets.puppet.com from the puppetlabs side. On the other side the encoding may have changed (it was authenticode signed but then that was removed).

File: CHANGELOG.md (hunk 2 above)
Comment (ferventcoder) [inline CHANGELOG.md]: I've updated this a bit.

**Comment by ferventcoder** [comment]
Ad hoc builds kicked off at https://jenkins-modules.puppetlabs.com/view/3.%20windows%20only/view/ad%20hoc/view/chocolatey/job/forge-windows_puppetlabs-chocolatey_init-manual-parameters_adhoc/21/

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,46 @@
 ## 2016-06-01 Unsupported Release 0.7.0
 
  * Manage sources - `chocolateysource` - see [MODULES-3037](https://tickets.puppetlabs.com/browse/MODULES-3037)
- * $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
- * Fix: puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
+ * Includes community module releases up to 1.2.1, plus these additional fixes:
+   * $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
+   * Fix: puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
```
Comment (glennsarti) [inline CHANGELOG.md]: 👍

**Pull request merged**